runtime-config.md

```diff
@@ -1,6 +1,12 @@
 ## Mount Configuration
 
-Additional filesystems can be declared as "mounts", specified in the *mounts* array. The parameters are similar to the ones in Linux mount system call. [http://linux.die.net/man/2/mount](http://linux.die.net/man/2/mount)
+Additional filesystems can be declared as "mounts", specified in the *mounts* object.
+Keys in this object are names of mount points from portable config.
+Values are objects with configuration of mount points.
+The parameters are similar to the ones in Linux mount system call.
+[http://linux.die.net/man/2/mount](http://linux.die.net/man/2/mount)
+Note, that only mount points which listed in portable configuration will be
+really mounted.
```

Comment:
This sentence should all be on one line (README's Markdown formatting).  And I'd end with “will actually be mounted”, because things without `config.json` listings will not be mounted in any sense, so “really” doesn't seem to fit.